include/crow/multipart.h

```diff
@@ -95,7 +95,14 @@ namespace crow
             {
                 size_t found = header.find("boundary=");
```

Comment:
Maybe move those kind of const even to line before as `constexpr`. Then you can avoid magic numbers (`strlen()`).

Reply:
I'm sorry I don't follow, can you elaborate? (it doesn't help that GH shows the code above the line you're commenting on)

Reply:
I mean something like `constexpr char text[] = "boundary=";`. Then you can use below `strlen(text)` instead of `9`.

Reply:
I'm guessing that will make `strlen(text)` a constant, correct?